apps/jobs/views.py

```diff
@@ -2775,3 +2775,46 @@ def update_submission_started_at(request, submission_pk):
         return Response(response_data, status=status.HTTP_200_OK)
     else:
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(["PATCH"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((JWTAuthentication, ExpiringTokenAuthentication))
+def change_submission_status_to_cancel(request, challenge_pk, submission_pk):
```

Comment:
I think `submission_pk` should be enough here?

Reply:
We only want the participants of the team to cancel the submission using UI. To check whether team has participated in the challenge we need challenge pk